packages/mgt-chat/src/components/ChatListItem/ChatListItem.spec.tsx

```diff
@@ -0,0 +1,106 @@
+// import React from 'react';
```

Comment:
Should this file be removed?

Reply:
I left it for ease of re-enabling later. When we tackle the unit testing story, this can be safely deleted after an approach is determined.